cli/upload.py

```diff
@@ -159,43 +165,42 @@ def _start_procs(upload_info: api.UploadInfo, xlsx: str) -> list:
 
 def _wait_for_upload(procs: list) -> bool:
     """
-    Waits for all subprocesses and prints their stderr streams.
+    Waits for all subprocesses and click.echos their stderr streams.
     Returns bool - if an error has occurred during any if uploads
     """
 
     finished = set()
     errored = False
+    progress_bar = "."
     while len(finished) != len(procs) and not errored:
-        for i, p in enumerate(procs):
 
+        click.secho(f"Uploading files ({len(finished)} / {len(procs)} done)", bold=True)
+        click.echo(progress_bar)
+        for i, p in enumerate(procs):
+            click.secho(f"(proc {i + 1}) ", nl=False, fg="bright_blue")
             if i in finished:
+                click.secho("uploaded ", nl=False, fg="green", bold=True)
                 continue
+            else:
+                click.echo("uploading ", nl=False)
 
             if p.poll() != None:
                 finished.add(i)
 
                 if p.returncode != 0:
                     errored = True
 
-            errline = p.stderr.readline()
+            click.echo(p.args[-2])
 
-            # skipping "large file" warnings
-            if errline.strip() in _IGNORED_WARN_LINES:
-                continue
+        time.sleep(1)
+        click.secho("~" * 50, dim=True)
 
-            if errline and len(errline) > 2:  # skipping '* ' spinner lines
+        progress_bar += "."
 
-                # changing output from always "[0/1 files]" - due to manual
-                # parallelization of gsutil to "[x/y files]"
-                if errline.split("]", 1)[0].endswith("/1 files"):
-                    errline = errline.split("]", 1)[1]
-                print(
-                    f"[{len(finished)}/{len(procs)} done] {i+1}: {p.args[-2]}: "
```

Comment:
this reported progress for large files, like
```
[85/101 done] 77: [ 1.2 GiB/ 3.2 GiB]  38% Done
[85/101 done] 77: [ 1.9 GiB/ 3.2 GiB]  59% Done
...
[85/101 done] 77: [ 3.1 GiB/ 3.2 GiB]  99% Done
```
why are we removing those? I'd think showing progress and its speed is good and should not be removed.
Open for discussion

Reply:
Oh, I didn't know that - should've checked with you first! That's definitely something we should keep.

Reply:
Wait, actually - won't the addition on line 193 still print out progress for large files? All I removed was reporting `[{len(finished)}/{len(procs)} done]` on every line (now it's reported once for every process polling iteration: see line 177).